tests/test_uaa.py

```diff
@@ -25,14 +25,18 @@ def test_get_uaa_public_key_with_no_config_set(self, mock_request):
 
     @requests_mock.mock()
     def test_get_uaa_public_key_server_error_response(self, mock_request):
+        """When the getting of the public key fails a HTTPError exception is raised.  This test, however,
+        assertIsNone because the exception is consumed, an exception is logged and then returns None."""
```

Comment:
Closing `"""` of a block comment should be on a new line. (PEP 257).

Turns out we don't have linting on tests, so I guess it's up to you whether or not you act on this.

Reply:
I'll change it.  Good spot ^^